brave/brave-core
# Added BraveVPNController to BrowserWindowFeatures

fix https://github.com/brave/brave-browser/issues/41973

BraveVPNController will control vpn behavior related with browser window.


Resolves



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch browser with clean profile on Windows/macOS
2. Click `Brave VPN` entry from app menu
3. Check vpn bubble is shown
